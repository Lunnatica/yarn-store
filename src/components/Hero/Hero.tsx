import {
  StyledContentContainer,
  StyledDescription,
  StyledTitle,
  StyledWrapper,
} from './StyledHero';

export const Hero: React.FC = () => {
  return (
    <StyledWrapper>
      <StyledContentContainer>
        <StyledTitle>Yarn Store</StyledTitle>
        <StyledDescription>
          Order your favorite products made using our unique yarn colors that everyone loves!
        </StyledDescription>
      </StyledContentContainer>
    </StyledWrapper>
  );
};
