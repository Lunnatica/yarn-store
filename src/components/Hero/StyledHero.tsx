import styled from 'styled-components';
import { getTextStyles } from '../../theme/typography';

const StyledWrapper = styled.header`
  color: var(--colors-blank);
  background-image: linear-gradient(227.4deg, #5240cb -3.02%, #3c2e92 79.84%);
  padding: 100px 0;
  height: 75vh;
  display: flex;
  align-items: center;
`;

const StyledContentContainer = styled.div`
  width: 954px;
  padding: 0 20px;
  margin-left: auto;
  margin-right: auto;
  box-sizing: content-box;
`;

const StyledTitle = styled.p`
  ${getTextStyles(600, '80px', 1)}
`;

const StyledDescription = styled.h1`
  color: var(--colors-spindle);
  ${getTextStyles(400, '18px', '28px')}
  max-width: 477px;
  letter-spacing: 2%;
`;

export { StyledContentContainer, StyledWrapper, StyledTitle, StyledDescription };
