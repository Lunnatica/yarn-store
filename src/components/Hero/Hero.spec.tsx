import { render } from '@testing-library/react';

import { Hero } from './Hero';

describe('Hero', () => {
  it('should match the snapshot', () => {
    const { container } = render(<Hero />);
    expect(container).toMatchSnapshot();
  });
});
