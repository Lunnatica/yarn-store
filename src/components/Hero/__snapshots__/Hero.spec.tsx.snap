// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hero should match the snapshot 1`] = `
<div>
  <header
    class="StyledHero__StyledWrapper-sc-19xd1yq-0 flLjYk"
  >
    <div
      class="StyledHero__StyledContentContainer-sc-19xd1yq-1 cVDaAR"
    >
      <p
        class="StyledHero__StyledTitle-sc-19xd1yq-2 jojgOh"
      >
        Yarn Store
      </p>
      <h1
        class="StyledHero__StyledDescription-sc-19xd1yq-3 dmuowE"
      >
        Order your favorite products made using our unique yarn colors that everyone loves!
      </h1>
    </div>
  </header>
</div>
`;
